Sources/Cookie.php

```diff
@@ -393,8 +393,8 @@ public static function setLoginCookie($cookie_length, $id, $password = '')
 				}
 
 				$alias_cookie = clone $cookie;
-				$alias_cookie->data[3] = $alias_cookie->domain = $domain;
-				$alias_cookie->data[4] = $alias_cookie->path = $path;
+				$alias_cookie->domain = $domain;
+				$alias_cookie->path = $path;
```

Comment:
```suggestion
				$alias_cookie->custom_data[3] = $alias_cookie->domain = $domain;
				$alias_cookie->custom_data[4] = $alias_cookie->path = $path;
```

Reply:
Ahh, I missed that.  It was data and I couldn't find data.  I totally missed custom_data

Reply:
Not your fault. I'm the one who made the mistake to begin with.